code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -216,15 +216,15 @@
 		if("pod_civ")
 			desc = "A sleek civilian space pod."
 		if("pod_black")
-			desc = "An all black space pod with no insignias."
+			desc = "A plain black space pod without any distinctive markings."
 		if("pod_mil")
-			desc = "A dark grey space pod brandishing the Nanotrasen Military insignia"
```

Comment:
The capitalization seemed off, since there's been no mention of the "Nanotrasen Military" from what I've seen.